### PR TITLE
UK Check VAT (Independent Publisher)

### PR DESCRIPTION
**Pull request by fordosa90**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by fordosa90** [comment]
image

image

File: independent-publisher-connectors/U.K. Government Check VAT/readme.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Prerequisites
+There are no prerequisites needed for this connector.
+
+## Obtaining Credentials
+This connector does not use authentication, so no credentials are needed.
+
+## Supported Operations
+### Get VAT registration
+Allows the retrieval of name and address of a VAT regstered company.
```
Comment (v-eisljami) [inline independent-publisher-connectors/U.K. Government Check VAT/readme.md:14]: Readme file should be free from grammar and spelling errors. Please review it.

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hi @fordosa90, I hope you are doing well. I want to follow up with you and let you know that your connector is in production deployment. this process will complete within 4-6 weeks.
Please let me know if you have any questions.
Thank you very much.

**Comment by Amjed-Ayoub** [comment]
Hi @fordosa90 I hope you are doing well. This is to inform you that your connector is in the final steps, in documentation deployment now. The process will be complete by the end of the week.
Please let me know if you have any questions.
Thank you very much.

**Pull request merged**